app/controllers/api/repos_controller.rb

```diff
@@ -1,18 +1,26 @@
-class Api::ReposController < ApplicationController
-    before_action :get_orgs, :check_org
-    def index
-      render json: { repositories: @organization.repositories}, status: :ok
-    end
-
-    private
-    def get_orgs
-        @organization = @user.organizations.where(name: params[:org_id]).first
-    end
-
-    def check_org
-      if !@organization
-        render json: { message: 'No organizations were found' }, status: :not_found
-      end
-    end
-end
+class Api::ReposController < ApplicationController
+    before_action :get_orgs, :check_org
+    def index      
+      repos = @organization.repositories
+      repos_json = JSON.parse(repos.to_json())
+
+      # add code_alerts field to repositories json
+      repos.zip(repos_json).each { |repo, repo_json|
+        repo_json["code_alerts"] = repo.code_alerts
```

Comment:
per your suggestion in Discord, change this to display boolean `code_alerts_exist` field just like you did for orgs